lib/git-lfs-s3/application.rb

```diff
@@ -83,11 +93,15 @@ def protected!
       status service.status
       body MultiJson.dump(service.response)
     end
-
+    
```

Comment:
Yeah there will have to be multiple upstream PRs. That'll allow me to split out the features from fixing whitespace, etc. Minimizing the diffs for the features.